bot/bot.py

```diff
@@ -49,6 +50,22 @@ def __init__(self, *args, **kwargs):
 
         self.stats = AsyncStatsClient(self.loop, statsd_url, 8125, prefix="bot")
 
+    async def _cache_allow_deny_list_data(self) -> None:
+        """Cache all the data in the AllowDenyList on the site."""
+        full_cache = await self.api_client.get('bot/allow_deny_lists')
+
+        for item in full_cache:
+            type_ = item.get("type")
+            allowed = item.get("allowed")
+            metadata = {
+                "content": item.get("content"),
+                "comment": item.get("comment"),
+                "id": item.get("id"),
+                "created_at": item.get("created_at"),
+                "updated_at": item.get("updated_at"),
+            }
```

Comment:
Can the API really return an empty dict for records? Are all these `get`s necessary?

Reply:
I like `get`'s. I never considered whether they were necessary. If you insist, I guess I'll go over them all and do that.

Reply:
I suppose that's a good habit. Kind of like using `"${}"` in shell scripts event when it's not necessary so I don't have to think about all the complicated rules for when it is/isn't necessary. I don't insist, you can leave them. I just think the `[]` is preferable when possible since it's more concise.

Reply:
Cleaned up in a23b273734ddee5fd082bda4fa14aebfff1317ca